apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -658,6 +762,327 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
     end
   end
 
+  describe "evaluates correctness of new piggybacks" do
+    @tag fixtures: [:processor_filled]
+    test "detects none if there is no piggybacks", %{processor_filled: processor} do
```

Comment:
do we have tests (or at least do the old tests that do) covering situations when there are valid piggybacks and this list is correctly empty?